.github/workflows/CI.yml

```diff
@@ -520,6 +488,10 @@ jobs:
         if: steps.e2etest-release.outputs.cache-hit != 'true'
         run: s3_bucket_name=aws-otel-collector-test upload_to_latest=0 bash tools/release/image-binary-release/s3-release.sh
 
+      - name: Build Image
```

Comment:
Think we can just use https://github.com/marketplace/actions/build-and-push-docker-images

Reply:
Agreed. Seems the [load tar involves in the CD](https://github.com/aws-observability/aws-otel-collector/blob/main/.github/workflows/CD.yml#L336-L337) so I should do all of this in one PR. Will post another PR for this step soon.